src/modules/src/cpx/cpx.c

```diff
@@ -126,8 +126,14 @@ static void cpx(void* _param) {
           }
         }
         break;
+      case CPX_F_APP:
+      {
+        messageReceive(cpxRx);
```

Comment:
There should likely be an API to register a callback for app messages or to receive packet from it. A similar things is done in CRTP with the [appchannel](https://github.com/bitcraze/crazyflie-firmware/blob/23af53b3aae324e8b8da95a29ab630dc30c2bc32/src/modules/interface/app_channel.h#L35). I feel like a similar API could be used there since it is a very similar concept.